ultra/ultra/scenarios/common/distributions.py

```diff
@@ -391,6 +391,200 @@
             "start_time": "default",  # any value or default for LANE_LENGTH / speed_m_per_s
         },
     },
+    # -------------------------------------------------- Intersection specific traffic distribution ------------------------------------------------
+    # The following traffic distributions [low-interaction, mid-interaction, high-interaction] are created to emphasize interaction between ego and
+    # social vehicles at the intersections (T or Cross). The key differences between these distributions and the {low, mid, high}-density are that they
+    # do not focus on interactions beyond the intersection, uses very limited number of social vehicles and there are no social vehicles in the ego
+    # mission route. In terms of implementation, these distribution will be used inside the simple level, where the goal is to make the ego agent
+    # familiar with the intersection
+    "low-interaction": {
+        "routes": {
+            "south-west": None,
+            "south-east": None,
+            "south-north": None,  # blocking
+            "west-east": {
+                "vehicles": 1,
+                "distribution": behavior_distribution,
+                "start_end_on_different_lanes_probability": 0.0,
+                "begin_time_init": {
+                    "func": burst_begin_time_init_func,
+                    "params": {
+                        "vehicle_cluster_size": (1, 1),
+                        "time_between_cluster": (5, 15),
+                        "time_for_each_cluster": 1,
+                    },
+                },
+                "has_turn": False,
+                "deadlock_optimization": True,
+                "pos_offsets": {
+                    "start": (160, 165),
+                    "end": (20, 20),
+                },
```

Comment:
Hmm, so these offsets are meant for the social vehicles (traffic), it specifies were the start/end points of the vehicle. I included them inside the distributions.py because that is where we are setting up the traffic and the social vehicles begin times and other traffic characteristics. It is actually something we can talk about in a meeting

Reply:
Okay sure, sounds good.